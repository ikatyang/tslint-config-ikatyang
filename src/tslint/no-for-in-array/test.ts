// good

[1, 2].forEach((_, index) => {

});

// bad

for (const key in [1, 2]) {

}
