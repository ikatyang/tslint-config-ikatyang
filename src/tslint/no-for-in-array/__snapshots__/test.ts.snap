// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

[1, 2].forEach((_, index) => {

});

// bad

for (const key in [1, 2]) {
~~~~~~~~~~~~~~~~~~~~~~~~~~~

~nil
}
~ [for-in loops over arrays are forbidden. Use for-of or array.forEach instead.]

`;
